kool-core/src/commonMain/kotlin/de/fabmax/kool/pipeline/backend/RenderBackend.kt

```diff
@@ -22,7 +22,7 @@ interface RenderBackend {
 
     val frameGpuTime: Duration
 
-    fun renderFrame(ctx: KoolContext)
+    suspend fun renderFrame(ctx: KoolContext)
```

Comment:
Since wgpu4k uses suspend for async work, it brings some breaking changes like this.

Reply:
I guess that's ok. Long running suspending functions shouldn't be called from `renderFrame` though, since that would obviously cause frame drops. But for things like buffer copies etc. which simply block on the other backends it should be fine.